simple/schema.py

```diff
@@ -147,7 +154,38 @@ class Gravities(Base):
     __tablename__ = 'Gravities'
     source = Column(String(100), ForeignKey('Sources.source', ondelete='cascade', onupdate='cascade'),
                     nullable=False, primary_key=True)
-    gravity = Column(Enum(Gravity))  # restricts to enumerated values
+    gravity = Column(Enum(Gravity), nullable=False)  # restricts to enumerated values
```

Comment:
why do we not want any of the above to be null? e.g. you can have sources with parallaxes and no proper motion in the same entry, no?

Reply:
Those are in different tables. This is saying that if you have an entry in the **Gravities** table, it should have a **gravity** value, otherwise you shouldn't be adding an entry here. It's OK for a source to have entries in one table, but not in another (such as parallaxes but no gravity measurements).